turbinia/workers/__init__.py

```diff
@@ -761,8 +761,15 @@ def execute(
               'empty new Evidence.'.format(evidence.name, evidence.source_path))
           result.log(message, level=logging.WARN)
         else:
-          result.add_evidence(evidence, self._evidence_config)
-
+          try:
```

Comment:
It would be nice if this exception message could be bubbled up into the task result `status` as well so it's easy to see from the client if there were no results.  Not sure of the best way to get that plumbed through though.   If `close` is True, then we could set it below, but not every task has that set.  It does look like the plaso task sets `close=True` when it calls this method though.  We might want to append it to the existing status though (by updating `result.status` directly), but I'm not sure if `close()` will overwrite that.

Reply:
This updates the status directly now.